docs/collector.physical_disk.md

```diff
@@ -0,0 +1,80 @@
+# physical_disk collector
+
+The physical_disk collector exposes metrics about physical disks
+
+|||
+-|-
+Metric name prefix  | `physical_disk`
+Data source         | Perflib
+Counters             | `physicalDisk` ([`Win32_PerfRawData_PerfDisk_physicalDisk`](https://msdn.microsoft.com/en-us/windows/hardware/aa394307(v=vs.71)))
+Enabled by default? | Yes
+
+## Flags
+
+### `--collector.physical_disk.disk-whitelist`
+
+If given, a disk needs to match the whitelist regexp in order for the corresponding disk metrics to be reported
+
+### `--collector.physical_disk.disk-blacklist`
+
+If given, a disk needs to *not* match the blacklist regexp in order for the corresponding disk metrics to be reported
+
+## Metrics
+
+Name | Description | Type | Labels
+-----|-------------|------|-------
+`requests_queued` | Number of requests outstanding on the disk at the time the performance data is collected | gauge | `disk`
+`read_bytes_total` | Rate at which bytes are transferred from the disk during read operations | counter | `disk`
+`reads_total` | Rate of read operations on the disk | counter | `disk`
+`write_bytes_total` | Rate at which bytes are transferred to the disk during write operations  | counter | `disk`
+`writes_total` | Rate of write operations on the disk  | counter | `disk`
+`read_seconds_total` | Seconds the disk was busy servicing read requests | counter | `disk`
+`write_seconds_total` | Seconds the disk was busy servicing write requests | counter | `disk`
+`free_bytes` | Unused space of the disk in bytes (not real time, updates every 10-15 min) | gauge | `disk`
+`size_bytes` | Total size of the disk in bytes (not real time, updates every 10-15 min) | gauge | `disk`
+`idle_seconds_total` | Seconds the disk was idle (not servicing read/write requests) | counter | `disk`
+`split_ios_total` | Number of I/Os to the disk split into multiple I/Os | counter | `disk`
```

Comment:
Can you add the prefix to the documented metrics? E.G. `windows_physical_disk_requests_queued`

Reply:
Done in 153656f41415eee5166cbbb182d33733a7f141b7 for #803.

Note: logical_disk collector is also missing the prefix.